Makefile

```diff
@@ -7,11 +7,11 @@ bench:
 
 .PHONY: build.bench
 build.bench:
-	tinygo build -o bench/case/case.wasm -scheduler=none -target=wasi bench/case/case.go
+	tinygo build -o bench/case/case.wasm -gc=leaking -scheduler=none -target=wasi bench/case/case.go
```

Comment:
This is a temporary workaround on the bug of TinyGo's conservative GC (enabled by default). I will raise an issue to TinyGo repo later.